.github/renovate.json

```diff
@@ -1,79 +1,124 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:recommended"
-  ],
-  "branchPrefix": "renovate/",
-  "baseBranches": ["master", "rhoai-2.16"],
-  "ignoreTests": true,
-  "automergeType": "pr",
-  "automerge": true,
-  "enabledManagers": ["dockerfile", "tekton", "rpm"],
-  "dockerfile": {
-    "schedule": ["at any time"],
-    "enabled": true,
-    "additionalBranchPrefix": "{{baseBranch}}/",
-    "branchPrefix": "konflux/mintmaker/",
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": ["config:recommended"],
+    "branchPrefix": "renovate/",
+    "baseBranches": ["main", "rhoai-2.16", "rhoai-2.17"],
```

Comment:
why isn't 2.13 on this list, after the branch got
* https://github.com/red-hat-data-services/kubeflow/pull/194